lib/teaspoon/deprecated.rb

```diff
@@ -83,6 +83,11 @@ def no_coverage(*)
         []
       end
       alias_method :no_coverage=, :no_coverage
+
+      def use_framework=(name, version = nil)
```

Comment:
Unused method argument - `version`. If it's necessary, use `_` or `_version` as an argument name to indicate that it won't be used.